webapp/src/utils/config.spec.ts

```diff
@@ -147,8 +147,53 @@ describe('LyraConfig', () => {
             '    path: anyValue',
           ].join('\n'),
         });
-        const readFromDirFunc = () => LyraConfig.readFromDir('/path/to/repo');
-        await expect(readFromDirFunc()).rejects.toThrow(LyraConfigReadingError);
+        const promise = () => LyraConfig.readFromDir('/path/to/repo');
+        await expect(promise).rejects.toThrow(LyraConfigReadingError);
+      });
+    });
+  });
+
+  describe('getProjectConfigByPath()', () => {
+    it('reads message kind and path from lyra.yml', async () => {
+      mock({
+        '/path/to/repo/lyra.yml': [
+          'projects:',
+          '- path: foo',
+          '  messages:',
+          '    format: yaml',
+          '    path: locale',
+          '  translations:',
+          '    path: locale',
+        ].join('\n'),
+      });
+      const config = await LyraConfig.readFromDir('/path/to/repo');
+      const projectConfig = config.getProjectConfigByPath('foo');
```

Comment:
I think the solution  is anti pattern, since we are going to make a private constructor a public one just to make it testable.
I think it is okay for now to have a wide scope unit test rather than do that